blitz.py

```diff
@@ -341,8 +348,10 @@ def get_char_radius(player_index: int = 10):
 def radius_movement(radius: int = 580, direction="forward"):
     player_coords = player_array[controlling_player()].get_coords()
     target_coords = [-400, -400]
-    player_coords[0] *= radius / get_char_radius(controlling_player())
-    player_coords[1] *= radius / get_char_radius(controlling_player())
+    char_radius = get_char_radius(controlling_player())
+    if char_radius != 0:
+        player_coords[0] *= radius / char_radius
```

Comment:
If there are only two things in `player_coords` we can do this as a list comprehensions:

```python
if char_radius != 0:
    player_coords = [coord * radius for coord in player_coords]
```
Slightly easier to read with less code duplication.

Reply:
It would be `coord * radius / char_radius for ...` I think. I'm not sure it's more easy to read, I'm not entirely friends with list comprehensions in Python :P

The main issue was that char_radius would sometimes be 0 (I think in between plays the characters are reset) and cause a divide-by-zero exception to be thrown.

Reply:
Yea, you are right, it would be `coord * radius / char_radius`. I think it's way easier to read personally, but that might be all the time I've been in the bowels of python. If you don't want to make the change, that's fine.